packs/chef/actions/chef_client.py

```diff
@@ -8,14 +8,14 @@ class ChefClientRunner(ChefRunner):
     ChefRunner type implementation.
     Invokes chef-client binary with given arguments.
     '''
-
-    chef_binary = 'chef-client'
```

Comment:
I personally prefer class variable (previous) approach since it's more clear in this case and in this case it also works fine.

It's also worth noting that class variable is not exactly the same as instance variable, but in this case either one works fine.